src/utils/join.js

```diff
@@ -1,7 +1 @@
-// For some reason path.posix.join is undefined in webpack
-// Also, this is just much smaller
-import { normalizePath } from './normalizePath.js'
-
-export function join(...parts) {
-  return normalizePath(parts.map(normalizePath).join('/'))
-}
+export { join } from 'path'
```

Comment:
This needs to be:

```
export { join } from 'path/posix'
```

(or whatever the equivalent is)

Reply:
@mojavelinux can you create a PR? This one is already merged. Also, it would be nice to have tests that fail before the fix.